doc/tutorial/scalablytyped.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: doc
+title: Integrate JavaScript libraries with ScalablyTyped
+---
+
+In this third tutorial, we learn how to integrate JavaScript libraries with [ScalablyTyped](https://scalablytyped.org/).
+
+We start here with the project developed in the previous tutorial about [UI development with Laminar](./laminar.html).
+To follow along this tutorial, either use the result of the previous tutorial, or checkout [the laminar-end-state branch](https://github.com/sjrd/scalajs-sbt-vite-laminar-chartjs-example/tree/laminar-end-state) of the accompanying repo.
+
+If you prefer to navigate the end result for this tutorial directly, checkout [the scalablytyped-end-state branch](https://github.com/sjrd/scalajs-sbt-vite-laminar-chartjs-example/tree/scalablytyped-end-state) instead.
+
+There are no playgrounds in [Scribble](https://scribble.ninja/) for this tutorial because it does not support ScalablyTyped.
+You will need a local project to follow along.
+
+## Prerequisites
+
+Make sure to install [the prerequisites](./index.html#prerequisites) before continuing further.
+
+## Set up ScalablyTyped with Chart.js
+
+We will use [Chart.js](https://www.chartjs.org/), a JavaScript library, to draw a bar chart out of the shopping list data.
+In order to get static types and bindings for Chart.js, we use [ScalablyTyped](https://scalablytyped.org/).
+ScalablyTyped can read TypeScript type definition files and produce corresponding [Scala.js facade types](/doc/interoperability/facade-types.html).
+
+We set up our new dependencies as follows.
+
+In `project/plugins.sbt`, we add a dependency on ScalablyTyped:
+
+{% highlight scala %}
+addSbtPlugin("org.scalablytyped.converter" % "sbt-converter" % "1.0.0-beta40")
+{% endhighlight %}
+
+In `package.json`, we add dependencies on Chart.js, its TypeScript type definitions, and on the TypeScript compiler, which is required by ScalablyTyped:
```

Comment:
Consider providing the relevant `npm` invocations rather than suggesting to change the file. IIRC, doing this does not deal well with lock files.